<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VotingApplication</class>
 <widget class="QMainWindow" name="VotingApplication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>225</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>225</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>275</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VOTING APPLICATION</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="VotingLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>VOTING APPLICATION</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="IDLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IDInput">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="CandidatesLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CANDIDATES</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="JaneVote">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>85</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Jane</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">VotingGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="JohnVote">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>105</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>John</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">VotingGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="SubmitButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>132</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>SUBMIT VOTE</string>
    </property>
   </widget>
   <widget class="QLabel" name="InfoLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>155</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Input ID and Vote</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>250</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="VotingGroup"/>
 </buttongroups>
</ui>
